.travis.yml

```diff
@@ -108,6 +108,18 @@ before_script:
   # https://github.com/kubernetes/kubernetes/issues/61058#issuecomment-372764783
   - sudo mount --make-shared / && sudo service docker restart
 script:
-  - ./scripts/ci/travis_script.sh
-after_success:
```

Comment:
@kaxil Let's make sure we're still doing codecov once the tests pass. Not sure if removing this would turn it off.